sdk/communication/communication-identity/rollup.test.config.js

```diff
@@ -0,0 +1,11 @@
+import { makeConfig } from "@azure/dev-tool/shared-config/rollup";
+import * as base from "./rollup.patched.config";
+
+const inputs = makeConfig(require("./package.json"));
+
+if (!process.env.ONLY_NODE) {
+  // replace the original test config with a patched one
```

Comment:
I'm not sure if we have a guideline on comments but I'd prefer a comment to start with a capital letter and end with a dot (.)
You may disregard this.